recipes/nd2reader/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "nd2reader" %}
+{% set version = "3.0.0" %}
+{% set sha256 = "18c721250669de804b85b7295e9885c94c21f99918b8da451ebbbab84fe29612" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >=1.6.2
+    - six >=1.4
+    - xmltodict >=0.9.2
+    - pims >=0.3.0
+  run:
+    - python
+    - numpy >=1.6.2
+    - six >=1.4
+    - xmltodict >=0.9.2
+    - pims >=0.3.0
+
+test:
+  imports:
+    - nd2reader
+
+about:
+  home: https://github.com/rbnvrw/nd2reader
+  license: GPL-3.0
```

Comment:
Please submit an issue or PR upstream to add a `MANIFEST.in` to the source repo that will force the `LICENSE.txt` file distributed with the package. 

For GPL, we must distribute the license, so until the above happens in a future release, please copy the `LICENSE.txt` file into the recipe directory and add the following:

```
license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE.txt'
```

See:
http://conda-forge.github.io/docs/meta.html#packaging-the-license-manually

and an example:

https://github.com/conda-forge/wsaccel-feedstock/tree/master/recipe